### ISSUE TITLE
`HandlersInfo` deprecation

### ISSUE THREAD

**Issue by raphaelns-developer** [ronco/ember-cli-meta-tags#78]
```
DEPRECATION: You attempted to use "_routerMicrolib.currentHandlerInfos" which is a private API that will be removed. [deprecation id: remove-handler-infos] See https://emberjs.com/deprecations/v3.x#toc_remove-handler-infos for more details.

```

"ember-source": "~3.7.0",
"ember-cli-meta-tags": "^5.3.0",

**Comment by snewcomer** [comment]
Hmmmm that would mean this conditional isn't working... 🤔 

https://github.com/ronco/ember-cli-meta-tags/blob/master/addon/services/head-tags.js#L26

**Comment by CvX** [comment]
Interesting… in my app, with ember-source 3.7.2 and ember-cli-meta-tags 5.3.0, I'm no longer getting this deprecation warnings.

**Comment by raphaelns-developer** [comment]
When i start the command `ember s` i receive this warning.

**Comment by puwelous** [comment]
me too

**Comment by bobisjan** [comment]
This can be closed #118 🥳
